src/WebJobs.Script.WebHost/FileMonitoringService.cs

```diff
@@ -245,15 +247,15 @@ private Task RestartAsync()
         {
             if (!_shutdownScheduled)
             {
-                _scriptEnvironment.RestartHost();
+                _scriptHostManager.RestartHostAsync();
```

Comment:
Should probably return this Task. I realize previously we were not awaiting as well, but if we return the Task, the caller can decide